d365fo.tools/functions/get-d365labelidvalid.ps1

```diff
@@ -0,0 +1,49 @@
+<#
+    .SYNOPSIS
+        Checks if a string is a valid 'Label Id' format
+ 
+    .DESCRIPTION
+        This function will validate if a string is a valid 'Label Id' format.
+
+    .PARAMETER LabelId
+        LabelId string to validate
+ 
+    .EXAMPLE
+        Get-D365LabelIdIsValid -LabelId "ABC123" #True
+        Get-D365LabelIdIsValid -LabelId "@ABC123" #True
+        Get-D365LabelIdIsValid -LabelId "@ABC123_1" #False
+        Get-D365LabelIdIsValid -LabelId "ABC.123" #False
```

Comment:
We would like to have each and every single example to be in a separate .EXAMPLE section.

This is both from a user perspective and from a testing perspective.

For each example section we would like that you describe what the example does and what to expect from it, written in a meaningful and descriptive way.

You can get some inspiration from most functions, but this one shows in a good way what we are trying to do:
https://github.com/d365collaborative/d365fo.tools/blob/development/docs/Get-D365Module.md